ApiClient/Source/Query.swift

```diff
@@ -27,12 +27,18 @@ import Foundation
 import Alamofire
 
 public protocol Query {
+    /// The type returned by the query.
     associatedtype Result
     
+    /// The url of the query.
     var url: URL { get }
+    /// The HTTP method of the query.
     var method: HTTPMethod { get }
+    /// The HTTP parameters of the query.
```

Comment:
Please go a bit more into detail what exactly this is about. The HTTP specs don't mention anything called "HTTP parameters" neither does our API documentation.
As someone writing a `Query` implementation, when should I put something here? What should I put here?